hw/ip/i2c/data/i2c.hjson

```diff
@@ -262,6 +264,11 @@
           desc: "Target mode receive FIFO is empty"
           resval: "1"
         }
+        { bits: "30"
+          name: "ACQDEPTH"
```

Comment:
I've left a comment alongside the assignment, suggesting that we may want to make all of the physical FIFO depths visible to software since we are going to end up with multiple deployments of the I2C block with different properties.
At present the ACQDEPTH field is just a single bit indicating whether this particular I2C block has a deepened ACQ FIFO (as is the plan for MCTP support).

Reply:
Can we please postpone this change to the PR that deepens the ACQ FIFO? It may be that all I2C instances have the same depth on the ACQ FIFO, in which case this wouldn't be needed. I think currently we don't have the information to make this decision yet.